tests/ai_safety/evalhub/test_evalhub_conversion.py

```
@@ -0,0 +1,225 @@
+def _get_resource(admin_client: DynamicClient, api_version: str):
+    """Get the DynamicClient resource handle for a specific EvalHub API version."""
+    return admin_client.resources.get(api_version=api_version, kind=EVALHUB_KIND)
+
+
+def _cleanup_evalhub(admin_client: DynamicClient, name: str, namespace: str) -> None:
+    """Delete an EvalHub resource, ignoring NotFound."""
+    try:
+        resource = _get_resource(admin_client, EVALHUB_FULL_API_VERSION_V1)
+        resource.delete(name=name, namespace=namespace)
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)